hc/front/tests/test_my_checks.py

```diff
@@ -58,3 +58,14 @@ def test_it_shows_amber_check(self):
 
         # Mobile
         self.assertContains(r, "label-warning")
+
+    def test_it_shows_nag_check(self):
```

Comment:
it's good practice to add docstring on tests so as they explain better what the test achieves